.github/workflows/build_deploy_wheels.yml

```diff
@@ -32,25 +38,27 @@ jobs:
           system-packages: 'gcc-gfortran'
           pre-build-command: 'export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib/; export SOLCORE_WITH_PDD=1'
 
-      - name: Publish wheels to PyPI
-        if: startsWith(github.ref, 'refs/tags')
-        env:
-          TWINE_USERNAME: '__token__'
-          TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
-        run: |
-          twine upload dist/*-manylinux*.whl --skip-existing
+#       - name: Publish wheels to PyPI
```

Comment:
You can uncomment these. This is not a tagged commit, so there is no risk of it being uploaded accidentally.